script/elevate-cpanel.PL

```diff
@@ -952,9 +952,7 @@ Update the current distro packages then reboot.
 
 sub run_stage_2 ($self) {
 
-    Elevate::Marker::startup();
-
-    Elevate::Motd->setup();
+    $self->run_component_once( 'Grub2' => 'verify_cmdline' ) if $self->should_run_leapp();
```

Comment:
TRIVIAL/PREFERENCE:  I think it would make the code easier to read if we executed this component no matter what.  Then, just have the first line of `verify_cmdline` be `return unless $self->should_run_leapp();`